vimeo-networking/src/main/java/com/vimeo/networking/model/playback/VideoLog.java

```diff
@@ -68,6 +75,15 @@ public String getPlayLoggingUrl() {
         return mPlayLink;
     }
 
+    public boolean isExitLinkEmpty() {
```

Comment:
Opportunity to just have a string method called `isEmpty`so this same logic isn't duplicated for each string (null check and isEmpty check)?